core/services/relay/evm/mercury/wsrpc/pool.go

```diff
@@ -220,11 +221,11 @@ func (p *pool) Ready() error {
 	if p.closed {
 		return errors.New("pool is closed")
 	}
-	return nil
+	return p.cacheSet.Ready()
```

Comment:
I suppose we don't need to forward this, since it comes through the health report :thinking: